pages/how-to/signature.md

```diff
@@ -23,12 +23,12 @@ Emails are our primary form of communication with clients. This tool will allow
 
   <!-- Work phone number & Personal number-->
   <p style="margin:0 0 8px 0;">
-    <a id="worknum" contenteditable="true" class="worknum" style="box-shadow:0 0 0 0 !important; font-size:14px; font-weight:200 !important; color:#333;font-family:Helvetica Neue, Helvetica, sans-serif; text-decoration:none; border-bottom:1px solid #BAB8B0;; padding-bottom:1px;" href="tel:+442035830972">+44 20 3583 0972</a><span style="color:#BAB8B0;" contenteditable="true">&nbsp;&nbsp;|&nbsp;&nbsp;</span><a class="directnum" id="directnum" contenteditable="true" class="number" style="box-shadow:0 0 0 0 !important;  font-size:14px; font-weight:200 !important; color:#333;font-family:Helvetica Neue, Helvetica, sans-serif; text-decoration:none; border-bottom:1px solid #BAB8B0; padding-bottom:1px;" href="tel:+441234567891">+44 12 3456 7891</a>
+    <a id="worknum" contenteditable="true" class="worknum" style="box-shadow:0 0 0 0 !important; font-size:14px; font-weight:200 !important; color:#333;font-family:Helvetica Neue, Helvetica, sans-serif; text-decoration:none; border-bottom:1px solid #BAB8B0; padding-bottom:1px;" href="tel:+442035830972">+44 20 3583 0972</a><span style="color:#BAB8B0;" contenteditable="true">&nbsp;&nbsp;|&nbsp;&nbsp;</span><a class="directnum" id="directnum" contenteditable="true" class="number" style="box-shadow:0 0 0 0 !important;  font-size:14px; font-weight:200 !important; color:#333;font-family:Helvetica Neue, Helvetica, sans-serif; text-decoration:none; border-bottom:1px solid #BAB8B0; padding-bottom:1px;" href="tel:+441234567891">+44 YOUR NUMBER</a>
```

Comment:
when you have long lines like these split them on several lines so git diffs become readable... eg

```

<a 
  id="worknum" 
  contenteditable="true" 
  class="worknum" 
  style="box-shadow:0 0 0 0 !important; 
             font-size:14px; 
             font-weight:200 !important; 
             color:#333;font-family:Helvetica Neue, Helvetica, sans-serif; 
             text-decoration:none;
              border-bottom:1px solid #BAB8B0; 
             padding-bottom:1px;" 
  href="tel:+442035830972">
    +44 20 3583 0972
</a>
<span style="color:#BAB8B0;" contenteditable="true">   
  &nbsp;&nbsp;|&nbsp;&nbsp;
</span>
...
```